libs/orders/frontend/view/src/lib/components/detail-order/detail-order.component.ts

```diff
@@ -0,0 +1,30 @@
+import { CommonModule } from '@angular/common';
+import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { IxModule } from '@siemens/ix-angular';
+
+import { orders } from '../orders';
+
+@Component({
+	selector: 'lib-detail-order',
+	standalone: true,
+	imports: [CommonModule, IxModule],
+	templateUrl: './detail-order.component.html',
+	styleUrl: './detail-order.component.scss',
+	encapsulation: ViewEncapsulation.None,
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DetailOrderComponent {
+	order = this.getOrder();
```

Comment:
Patrick and I did it with a function because  it complained that it might be undefined otherwise

Reply:
exactly the same as here: libs/facilities/frontend/view/src/lib/components/detail/detail.page.ts

Reply:
see other side